internal/configs/virtualserver_test.go

```diff
@@ -1960,3 +1961,44 @@ func TestGenerateEndpointsForUpstream(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateSlowStartForPlusWithInCompatibleLBMethods(t *testing.T) {
+	name := "test-slowstart-with-incompatible-LBMethods"
+	upstream := conf_v1alpha1.Upstream{Service: name, Port: 80, SlowStart: "10s"}
+	expected := ""
+
+	for lbMethod, _ := range incompatibleLBMethodsForSlowStart {
+		nlbMethod := generateLBMethod(upstream.LBMethod, lbMethod)
+		result := generateSlowStartForPlus(upstream, nlbMethod)
+
+		if !reflect.DeepEqual(result, expected) {
+			t.Errorf("generateSlowStartForPlus returned %v, but expected %v", result, expected)
+		}
+	}
+}
+
+func TestGenerateSlowStartForPlus(t *testing.T) {
+	name := "test-slowstart"
+
+	tests := []struct {
+		upstream conf_v1alpha1.Upstream
+		expected string
+	}{
+		{
+			upstream: conf_v1alpha1.Upstream{Service: name, Port: 80, SlowStart: "", LBMethod: "least_conn"},
+			expected: "",
+		},
+		{
+			upstream: conf_v1alpha1.Upstream{Service: name, Port: 80, SlowStart: "10s", LBMethod: "least_conn"},
+			expected: "10s",
+		},
+	}
+
+	for _, test := range tests {
+		lbMethod := generateLBMethod(test.upstream.LBMethod, "least_conn")
```

Comment:
there is no need to call `generateLBMethod`. I suggest making the method the third field of the struct you use in tests array.